lib/ansible/module_utils/facts/system/pkg_mgr.py

```diff
@@ -41,6 +41,7 @@
             {'path': '/usr/sbin/sorcery', 'name': 'sorcery'},
             {'path': '/usr/bin/installp', 'name': 'installp'},
             {'path': '/QOpenSys/pkgs/bin/yum', 'name': 'yum'},
+            {'path': '/usr/bin/rpm-ostree', 'name': 'rpm_ostree'},
```

Comment:
This won't work as `/usr/bin/rpm-ostree` has an entry at the start for `atomic_container`.